src/Polysemy/Error.hs

```diff
@@ -261,35 +262,42 @@ instance (Typeable e) => X.Exception (WrappedExc e)
 --
 -- @since 1.2.0.0
 errorToIOFinal
-    :: ( Typeable e
+    :: forall e r a
+    .  ( Typeable e
```

Comment:
Once you've converted to using `Any`, this `Typeable` constraint can be dropped.